### PR TITLE
changes.h for #2403

### PR DESCRIPTION
**Pull request by ESeNonFossiIo**
**changes.h** for [#2403](https://github.com/dealii/dealii/pull/2403)

### CODE REVIEW

File: doc/news/changes.h [hunk 1]
```diff
@@ -110,7 +110,17 @@ inconvenience this causes.
 <a name="specific"></a>
 <h3>Specific improvements</h3>
 
-<ol>
+<ol> 
+ <li> New: Add remove_hanging_nodes and remove_anisotropy in
```
Comment (luca-heltai) [inline doc/news/changes.h]: if you write these as `GridTools::remove_hanging_nodes()` and `GridTools::remove_anisotropy()`, Doxygen will automatically link this entry to the correct place in the documentation. Same below.

File: doc/news/changes.h [hunk 2]
```diff
@@ -110,7 +110,17 @@ inconvenience this causes.
...
+ GridTools. remove_hanging_nodes detects cells with hanging nodes and
+ refines the neighbours in the direction that removes hanging nodes or
+ in everery directions. remove_anisotropy refines a mesh
+ anisotropically such that the resulting mesh is composed by cells
```
Comment (luca-heltai) [inline doc/news/changes.h]: maybe `until the resulting mesh is composed by cells with ratio between the extension in each coordinate direction lower than a fixed value`.

File: doc/news/changes.h (hunk 1 above)
Comment (ESeNonFossiIo) [inline doc/news/changes.h]: done

File: doc/news/changes.h (hunk 2 above)
Comment (ESeNonFossiIo) [inline doc/news/changes.h]: thanks :-)

File: doc/news/changes.h [hunk 3]
```diff
@@ -110,7 +110,18 @@ inconvenience this causes.
 <a name="specific"></a>
 <h3>Specific improvements</h3>
 
-<ol>
+<ol> 
+ <li> New: Add GridTools::remove_hanging_nodes() and 
+ GridTools::remove_anisotropy() in GridTools. GridTools::remove_hanging_nodes() 
+ detects cells with hanging nodes and refines the neighbours in the direction 
+ that removes hanging nodes or in everery directions.
```
Comment (luca-heltai) [inline doc/news/changes.h]: everery -> every

**Pull request merged**